assembly/assembly-wsmaster-war/src/test/java/org/eclipse/che/WsMasterModuleTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che;
+
+import com.google.inject.Binder;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import com.google.inject.Module;
+import com.google.inject.name.Names;
+import io.micrometer.core.instrument.internal.TimedExecutorService;
+import java.util.concurrent.ExecutorService;
+import org.eclipse.che.api.deploy.MetricsOverrideBinding;
+import org.eclipse.che.api.deploy.jsonrpc.CheJsonRpcWebSocketConfigurationModule;
+import org.eclipse.che.api.deploy.jsonrpc.CheMajorWebSocketEndpointConfiguration;
+import org.eclipse.che.core.metrics.MetricsModule;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class WsMasterModuleTest {
```

Comment:
Do you think we need this test class?
If yes, then I think `setup` should be part of `shouldInjectMettered` because it not general setup for `WsMasterModule` I think